docs/src/CommutativeAlgebra/ca.md

```diff
@@ -494,9 +510,22 @@ codim(I)
 
 ## Affine Algebras
 
-By definition, an affine algebra over a field $K$, also called a $K$-algebra,
-is the quotient $A=R/I$ of a multivariate polynomial ring $R$ over $K$
-modulo a radical ideal $I$. 
+An affine algebra over a field $K$, or a $K$-algebra, is the quotient $A=R/I$ of a multivariate
```

Comment:
To me, a $K$-algebra is a much more general concept (in particular, it may be non-commutative). Do we really want to say this here?

Reply:
Das ist nur ein Typo: das Wort affine fehlt vor algebra.